### DISCUSSION TITLE
Voice prompts: required or optional?

### DISCUSSION THREAD

**Discussion by ethanblake4** [PullDating/Org#118] [Ideas & Features]
Title

**Comment by WilliamDormer** [comment]
Idk, I like the idea of mandatory participation to get buy-in, but it may also drive users away from the platform.

**Comment by Oshyan** [comment]
Again I'm gonna say I'm really not in favor of making much about profiles mandatory, or forcing a lot of behavior. What I hope we can do instead is *encourage* and *promote* good behavior. This feels far more aligned with our mission, and less presumptuous as well.

So in this case we simply preferentially show and/or highlight profiles that *do* have audio prompts, rather than forcing everyone to have them. It's not that other profiles won't show up, they'll show just fine, but profiles with audio (and/or with a lot of photos, long description, etc.) will show *more* and be more visible or eye-catching in some way.

More on this here: #134

**Comment by shootingcharlie8** [comment]
I agree w/ @Oshyan on this - I dont want to be forced to make a voice prompt. This will also likely lead to low-quality responses or dead air, which would act to detract from the purpose anyways. I think we just need to encourage and make it clear that completing a voice prompt will (along w/ the rest of your profile) will make your profile will make it more visible in the queue.
